core/src/test/java/com/tweet/core/service/impl/TweetServiceImplTest.java

```diff
@@ -0,0 +1,77 @@
+package com.tweet.core.service.impl;
+
+import com.tweet.core.model.Tweet;
+import com.tweet.core.service.TweetService;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class TweetServiceImplTest {
+
+    private static final LocalDate NOW = LocalDate.now();
+    private static TweetService tweetService = new TweetServiceImpl();
```

Comment:
I am not sure if that will actually work. Service have autowire of repository. 
This implies rather integration test instead of simple unit one.

To make it work correctly i would suggest to add

@RunWith(SpringRunner.class)
@SpringBootTest
class TweetServiceImplTest {

and 
 @Autowired
    private TweetService tweetService;

Reply:
and make this class public